src/repos.json

```diff
@@ -54,6 +54,8 @@
     { "language": "ruby", "repo": "GoogleCloudPlatform/ruby-docs-samples"},
     { "language": "ruby", "repo": "GoogleCloudPlatform/getting-started-ruby"},
     { "language": "ruby", "repo": "googleapis/ruby-spanner-activerecord", "isTeamIssue": true},
+    
+    { "language": "yaml", "repo": "GoogleCloudPlatform/workflows-samples"},
```

Comment:
Thanks!  Should all of the issues in that repository belong to one subteam?  If so, mind adding a line to `teams.json` as well?  Otherwise these will show up assigned to yoshi.

Reply:
Alright, added a line to `teams.json`.